ui/Rhino/AGS/dev/AGS_form_from_layer_cmd.py

```diff
@@ -20,10 +20,12 @@ def RunCommand(is_interactive):
 
     scene = sc.sticky['AGS']['scene']
 
-    layer = compas_rhino.rs.CurrentLayer()
-    layer_name = compas_rhino.rs.GetString("Layer to construct FormDiagram", layer)
+    layer_name = compas_rhino.rs.GetString("Type name of layer  with lines to construct FormDiagram")
```

Comment:
to be honest i would just call this `layer`. less underscores make the code more readable...